libnavigation-base/src/test/java/com/mapbox/navigation/base/route/RouteUrlTest.kt

```diff
@@ -52,16 +51,29 @@ class RouteUrlTest {
 
     @Test
     fun checkQueries() {
-        val token = "pk_token1212.dsda"
+        val token = "pk.token1212.dsda"
         val routeUri = setupRouteUrl(
             accessToken = token,
             steps = true,
             geometries = RouteUrl.GEOMETRY_POLYLINE,
             overview = RouteUrl.OVERVIEW_SIMPLIFIED,
             voiceIntruction = false,
+            voiceUnit = RouteUrl.METRIC,
             bannerIntruction = true,
             roundaboutExits = true,
-            enableRefresh = false
+            enableRefresh = false,
+            continueStraight = false,
+            exclude = DirectionsCriteria.EXCLUDE_MOTORWAY,
+            language = "en",
+            bearings = "20.0;45.0;50.0;90.0",
```

Comment:
This https://github.com/mapbox/mapbox-navigation-android/pull/2551#discussion_r390291514 still applies. Same for `waypointNames` for example.

One thing that I'm also missing here is that we're not checking the `coordinates` in this test and we should definitely add it cc @korshaknn

Reply:
@Guardiola31337 coordinates checking [here](https://github.com/mapbox/mapbox-navigation-android/pull/2551/files#diff-4171f48ef05fb04714b5122423a92dd3L26)

Please take attention that this test checks params that are passed to `RouteUrl` are applied and exist in url. If `language = "en"` has replaced by `language = "no language"`, tests would pass. And it's ok, because test is not checking it

Reply:
> coordinates checking [here](https://github.com/mapbox/mapbox-navigation-android/pull/2551/files#diff-4171f48ef05fb04714b5122423a92dd3L26)

Why not here too? As the rest of the fields? 🤔 

> Please take attention that this test checks params that are passed to `RouteUrl` are applied and exist in url. If `language = "en"` has replaced by `language = "no language"`, tests would pass. And it's ok, because test is not checking it

Yeah, I know. I suggested it so that the test is more meaningful, especially if you look at it as an outsider, those values wouldn't work in real life so IMO we should reflect what is going to work in production to avoid confusion.

Reply:
yeah agree, but if any of field is changed for sample `comma` replaced by `semicolon` it shouldn't reflect in tests so IMO I should write a note, that params is not necessary the same as in production code, wdyt?